src/main/scala/PcapExample.scala

```diff
@@ -2,13 +2,7 @@
 package com.scalawilliam.scalanative
 
 import scala.scalanative.native
-import scala.scalanative.native.{
-  CString,
-  CUnsignedInt,
-  Ptr,
-  fromCString,
-  toCString
-}
+import scala.scalanative.native._
```

Comment:
Sorry I tried, but I think it might be easier if you did that.  Installing lxml with pip doesn't seem to work on my system nor in docker.  However,  I've updated the source line numbers so that they'll be correct once again.